ANSWERS.md

```diff
@@ -1,4 +1,19 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+
+This is a trick, or outdated question. As of the React Hooks implementation There is no particular specification that increases the viability of a class component over a functional component. A functional component is in many cases less resources intensive, and requires a reduced code base to accomplish the same outcomes in compairson to the class componant.
```

Comment:
We talked about the question and answer. I'll try to remember to ask this question to be reworded.